nautobot/core/views/generic.py

```diff
@@ -67,6 +67,7 @@ class ObjectView(ObjectPermissionRequiredMixin, View):
 
     queryset = None
     template_name = None
+    use_new_ui = True
```

Comment:
Should this be in `ObjectListView` and other read-only mixins for `NautobotUIViewSet`?